src/main/java/org/hydev/mcpm/client/installer/InstallInteractor.java

```diff
@@ -84,7 +96,19 @@ public InstallResult installPlugin(InstallInput installInput)
                 installPlugin(dependencyInput);
```

Comment:
The recursive step doesn't use any results from the recursion. For example, if a dependency fails to be installed, the overall result would still return success if the initial plugin is a success. I can approve if we make this an issue and fix it later